templates/interface/Debian.erb

```diff
@@ -9,7 +9,7 @@ allow-hotplug <%= @interface %>
 <% if @manage_address -%>
     address <%= @manage_address %>
 <% end -%>
-<% if @manage_hwaddr -%>
+<% if @manage_address and ! @manage_address.empty? -%>
```

Comment:
ohh. crap... I updated this pr the wrong way. Xcuse me! line 12 should be at line 9. I'll update my pr strait away.